### DISCUSSION TITLE
another animation export formats

### DISCUSSION THREAD

**Discussion by hacknorris-code** [Orama-Interactive/Pixelorama#658] [Ideas]
so yes. now exporting animation from pixelorama can be only as GIF. but there are also other formats which can be animated. list:

- png (means - apng but saves also as png. can be animated)
- mp4 (might be deaf but still animated)
- avi 
- svg (yes it could also be added to export normal images)
- mkv (matroska)
- webp
- avif
- mng
- and some more video formats (too lazy to google)

also - these formats can be used to export static images too !!! (who will block you from just adding one frame as all the vid? will be static...)

waiting for update !!! 

=]

**Comment by OverloadedOrama** [comment]
Video exporting has been implemented in #980 and will be included in version 1.0, although they will require FFMPEG to be installed. Animated pngs has been available since version 0.11, and svg and webp are tracked in #968 and #746 respectively, so I will go ahead and close this proposal now.
